### PR TITLE
More theorems about the dimension of vector spaces

### PR DESCRIPTION
**Pull request by tirix**
This completes #3185 with more theorems about the dimension of vector spaces.
Especially:
- the dimension of a subspace is less than the dimension of the main space (`~lssdimle`)
- the dimension of a subspace plus the dimension of its left coset equals the dimension of the main vector space (`~qusdimsum`)

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -467236,6 +467245,280 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
...
+      ABCVMOZSWRWTWJWEWSXAWJWETWRWSWKNXKWJWEVOABXBXLAXDXFXCHIXEBCXERVAVCAXDCVPS
+      XLXBUOHCVQCVRVSVTWRWSWTXAWKWJCEBGWJRWKRWAVEWBAUAPZWLSZWRESZWSESZWFZTJEWJW
+      LDUGOZWMWRWNBXMCDWSGWJRWLRWMRAXDXQHVIAXFXQIVIAXNXOXPWCFXRRWNRAXNXOXPWDAXN
+      XOXPWGWHHWI $.
+
+    ${
+      $d V k u v x y z $.  $d F k u v y z $.  $d G k u v x y z $.
+      $d M k u v x y z $.  $d N k u v x y z $.  $d ph k u v x y z $.
+      quslmhm.f $e |- F = ( x e. V |-> [ x ] ( M ~QG G ) ) $.
+      $( If ` G ` is a normal subspace in ` M ` , then the "natural map" from
```
Comment (savask) [inline set.mm:467503]: What is a "normal subspace"? There's a similar notion from topology, but it's not relevant.
Did you mean a "submodule" (that is required in order to have a factor-module correctly defined)?

Also the last sentence probably should be "... is a left module homomorphism from **M** to M / G ."

File: set.mm [hunk 2]
```diff
@@ -467676,6 +468012,32 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
       YKZVWPUYHVVJXUSUYDUYIUYMVUGEUYKUBVWTUYERYTYLYMUYFUYG $.
   $}
 
+  ${
+    $d U x $.  $d W x $.  $d Y x $.
+    qusdimsum.x $e |- X = ( W |`s U ) $.
+    qusdimsum.y $e |- Y = ( W /s ( W ~QG U ) ) $.
+    $( Let ` W ` be a vector space, and let ` X ` be a subspace.  Then the
+       dimension of ` W ` is the sum of the dimension of ` X ` and the
+       dimension of the coset of ` X ` .  First part of theorem 5.3 in [Lang]
```
Comment (savask) [inline set.mm:468021]: Not a "coset", but a "factor space" (or a "quotient space" if you want to be consistent with your earlier comment, but Lang uses "factor space").

**Review by savask — COMMENTED** [review]
Great work! I think I spotted one typo and one misuse of the terminology.

My traditional remark is "please minimize". For example, the proof of `qusker` can be reduced from 1233 bytes to 1122 (other theorems will also benefit from minimization).

File: set.mm (hunk 1 above)
Comment (tirix) [reply set.mm:467503]: You're right, thanks! I've fixed that in the last commit.

**Comment by savask** [comment]
@tirix I think you added some .mmt files to the pull request.

**Comment by tirix** [comment]
> @tirix I think you added some .mmt files to the pull request.

Oh thanks, these will be part of the next PR!

**Pull request merged**